components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/services/TemplateManagerService.java

```diff
@@ -91,7 +96,8 @@ public void createBusinessRuleFromTemplate(BusinessRuleFromTemplate businessRule
                 saveBusinessRuleDefinition(businessRuleUUID, businessRuleFromTemplate, isDeployed); // todo: implement method
             } catch (TemplateManagerException e) {
                 // Saving definition is unsuccessful
-                log.error("Error in saving the Business Rule definition", e); // Exception is thrown from the saveBusinessRuleDefinition method itself
+                log.error("Error in saving the Business Rule definition", e);
+                // Exception is thrown from the saveBusinessRuleDefinition method itself
             }
         } catch (Exception e) {
             log.error(e.getMessage(), e);
```

Comment:
Give context and why catching exception ?

Reply:
FIxed by 9cf9a377bbb4bb9a29b5e04ffc9df4b605f84b44 and df3bc5a752f0dfcd03393c5f62409814722e720b